src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/handlers/CcdCallbackHandler.java

```diff
@@ -134,12 +134,17 @@ private String stringJoin(List<String> messages) {
 
         setUnsavedFieldsOnCallback(callback);
 
+        FormType formType = callback.getCaseDetails().getCaseData().getFormType();
         Map<String, Object> appealData = new HashMap<>();
         sscsDataHelper.addSscsDataToMap(appealData,
             callback.getCaseDetails().getCaseData().getAppeal(),
             callback.getCaseDetails().getCaseData().getSscsDocument(),
             callback.getCaseDetails().getCaseData().getSubscriptions(),
-            callback.getCaseDetails().getCaseData().getFormType());
+            formType);
+
+        if (formType != null && formType.equals(FormType.SSCS2)) {
+            appealData.put("childMaintenanceNumber", callback.getCaseDetails().getCaseData().getChildMaintenanceNumber());
```

Comment:
I think the pattern normally is to add to `sscsDataHelper.addSscsDataToMap()` as that is used by a few places. That way you only need to add it to one place rather than here and `SscsCaseTransformer.transformData()`

Reply:
I have added child maintenance

Reply:
You should be able to remove this block (from line 147 - 149) now

Reply:
Removed